### PR TITLE
Feature/imu test

### PR DESCRIPTION
**Pull request by HannesSommer**
@ZacharyTaylor , this should work better for the accelerometer

### CODE REVIEW

File: oomact/src/model/sensors/Imu.cpp [hunk 1]
```diff
@@ -253,10 +268,10 @@ void Imu::addMeasurementErrorTerms(CalibratorI & calib, const EstConf & /*ec*/,
     addImuErrorTerms(
         calib, *this, accelerometerName, measurements_->accelerometer,
         [&, this](const Timestamp timestamp, const AccelerometerMeasurement & m){
-          auto robot = calib.getModelAt(*this, timestamp, 2, {false});
+          auto modelAt = calib.getModelAt(*this, timestamp, 2, {false});
           return boost::make_shared<ErrorTermAccelerometer>(
-              robot.getAcceleration(inertiaFrame, getParentFrame()),
-              getTransformationExpressionTo(robot, inertiaFrame).toRotationExpression().inverse(),
+              modelAt.getAcceleration(getFrame(), inertiaFrame),
```
Comment (ZacharyTaylor) [inline oomact/src/model/sensors/Imu.cpp:305]: Was this where the error was?

Comment (HannesSommer) [reply oomact/src/model/sensors/Imu.cpp:305]: This was the first bug, the one within the Imu, discovered quite early. However, the real big problem was this https://github.com/ethz-asl/oomact/pull/32/commits/59231c71b07b4bc62675d48d631a58bd1246441a small change. Effectively this lets the same CoordinateFrames (bad name; they do the automatic kinematic) be generated but in flipped order -> the parent child relation gets inverted. This is only visible when you have more then two frames in a chain - and that wasn't used before the imu would use its own sensor frame connected to the body frame (static (non trajectory) links are now also supported (so far only for the sensors but easy to extend). As I said the FrameGraphModel is very new..

Comment (HannesSommer) [reply oomact/src/model/sensors/Imu.cpp:305]: Do you have a better idea for the naming? (CoordinateFrame = nodes in a kinematic chain)

**Comment by HannesSommer** [comment]
@ZacharyTaylor , I'd like to rewrite this branches history (cleaning up a bit before merging). Would that be an issue for you?

**Comment by ZacharyTaylor** [comment]
I have been merging it into the euroc example but we have been working in separate files so it should be fine

**Pull request closed without merging**